setup.py

```diff
@@ -0,0 +1,27 @@
+
+from setuptools import setup
+
+setup(
+    name='csc_510_hw1',
+    version='0.1',
+    description='Learning how to build a cool github repo', 
+    author='Harsh Kachhadia',
+    author_email='hmkachha@ncsu.edu',
+    url='https://github.com/dangoslen/csc_510_hw1',
+
+	  packages=find_packages(include=['code', 'code.*','data']),
+    long_description="""Learning how to build a cool github repo""",
+    classifiers=[
+          "License :: MIT License",
+          "Programming Language :: Python",
+          "Development Status :: 3 - Alpha",
+          "Intended Audience :: Developers",
+          "Topic :: cool github repo",
+      ],
+    keywords='csc-510 hw1 csc-510-hw1',
+    license='MIT License',	
+    install_requires=['numpy'], #numpy is just for sample
```

Comment:
I think we should put `pytest` in here as well since we use use it.. But I might mis-understand the purpose of this variable... do you have any good docs for it?

Reply:
Yes, the install-requires variable is for the user when he/she downloads our package. User doesn't need to run tests for the packages like developers. So, with that reduced usage, **pytest** should only be written under the **test_requires** section.
For further reference, you can read this article: [A Practical Guide to Using Setup.py](https://godatadriven.com/blog/a-practical-guide-to-using-setup-py/)